components/concept/engine/src/test/java/mozilla/components/concept/engine/EngineSessionTest.kt

```diff
@@ -447,4 +448,6 @@ open class DummyEngineSession : EngineSession() {
     fun notifyInternalObservers(block: Observer.() -> Unit) {
         notifyObservers(block)
     }
+
+    override fun captureThumbnail(): Bitmap? = null
```

Comment:
We could add some more tests here as well to keep the code coverage consistent, but since we've covered the most important parts already, I'm not picky.

Reply:
Done!